client/src/views/statefulSet.tsx

```diff
@@ -14,14 +14,29 @@ import PodCpuChart from '../components/podCpuChart';
 import getMetrics from '../utils/metricsHelpers';
 import {filterByOwner} from '../utils/filterHelper';
 import ContainersPanel from '../components/containersPanel';
-import {defaultSortInfo} from '../components/sorter';
+import {defaultSortInfo, SortInfo} from '../components/sorter';
 import ReplicasChart from '../components/replicasChart';
 import ChartsContainer from '../components/chartsContainer';
+import {ApiItem, K8sEvent, Metrics, Pod} from "../utils/types";
+
+type Props = {
+    namespace: string;
+    name: string;
+}
+
+type State = {
+    item?: ApiItem<any, any>;
```

Comment:
I think the type here should actually be `StatefulSet`, no?

Reply:
Yeah, good catch!